core-bundle/tests/Contao/DcaLoaderTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Contao;
+
+use Contao\Config;
+use Contao\CoreBundle\Tests\TestCase;
+use Contao\DcaLoader;
+use Contao\System;
+use Symfony\Component\Filesystem\Filesystem;
+
+class DcaLoaderTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $container = $this->getContainerWithContaoConfiguration($this->getTempDir());
+        System::setContainer($container);
+    }
+
+    protected function tearDown(): void
+    {
+        unset($GLOBALS['TL_DCA'], $GLOBALS['TL_MIME']);
+
+        $this->resetStaticProperties([System::class, Config::class, DcaLoader::class]);
+
+        parent::tearDown();
+
+        (new Filesystem())->remove($this->getTempDir());
```

Comment:
This should not be necessary:

https://github.com/contao/contao/blob/78abdd8cfd0520b960d12a1a6d9591e4709d56d8/test-case/src/ContaoTestCase.php#L47-L49

Reply:
Removed in bf799faaeba49ea39d3e9ad7e2f2d4c0e769ec70